docs/config.rst

```diff
@@ -145,6 +145,17 @@ Those configuration options are documented below:
             includePaths: [/https?:\/\/getsentry\.com/, /https?:\/\/cdn\.getsentry\.com/]
         }
 
+.. describe:: collectWindowErrors
+
+  By default, Raven collects all uncaught errors that have been caught by the `onerror` handler.
+  You can disable this behaviour by setting this flag to `false`. This may be useful when you only want to manually handle errors by using the `captureMessage` and `captureException`.
```

Comment:
> by using the `captureMessage` and `captureException`.

Drop unnecessary `the`. Should just be "by using `captureMessage` and `captureException`".